basic/exercise7/script7.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+function getNumberOfDaysUntilNextChristmas(date) {
+    var oneDayInMiliseconds = 1000 * 60 * 60 * 24;
+    var myDate = new Date;
```

Comment:
New date not declared incorrectly - you ignored my previous review and skype message.